### ISSUE TITLE
Add explanatory usage notebook

### ISSUE THREAD

**Issue by impredicative** [oval-group/smooth-topk#2]
Thanks for the paper and code. Is it possible to provide an explanatory Jupyter notebook that shows how to use these loss functions?

**Comment by lberrada** [comment]
Hi, I don't work with Jupyter notebooks but here is some minimal python code to use the loss function:
```python
import torch

from losses.svm import SmoothTopkSVM

# hyper-parameters
topk = 10  # top-10 classification
alpha = 1  # margin parameter of hinge loss, 1 is a default value
tau = 0.1  # smoothing temperature parameter (see paper for details)

# random data
batch_size = 3
n_dim = 20
n_classes = 100
x = torch.randn(batch_size, n_dim)
y = torch.randint(0, n_classes, size=(batch_size,)).long()

# model
model = torch.nn.Linear(n_dim, n_classes)

# loss function
loss_fn = SmoothTopkSVM(n_classes, alpha=alpha, tau=tau, k=topk)

# forward pass
loss = loss_fn(model(x), y)

# backward pass
loss.backward()

# print gradient
print(model.weight.grad)
```
